### PR TITLE
[ko] fixed Wrong Korean translations

### PR DESCRIPTION
**Pull request by suhhee1011**
This Pull request is about fixed issue #29027.

I changed the Wrong translation in 3 files.

1. ko/docs/concepts/workloads/pods/pod-lifecycle.md
2. ko/docs/tutorials/stateful-application/basic-stateful-set.md
3. ko/docs/concepts/scheduling-eviction/taint-and-toleration.md

Please let me know if there is more place to edit the wrong translation.

### CODE REVIEW

**Comment by jihoon-seo** [comment]
@suhhee1011 Thank you for your interest and contribution for this project!

보통 assign은 reviewer/approver 를 하는 것 같아서, @suhhee1011 님 자신을 assign 하신 것은 해제하겠습니다. 😊
/unassign @suhhee1011 

Previews:
- https://deploy-preview-30287--kubernetes-io-main-staging.netlify.app/ko/docs/concepts/scheduling-eviction/taint-and-toleration/
- https://deploy-preview-30287--kubernetes-io-main-staging.netlify.app/ko/docs/concepts/workloads/pods/pod-lifecycle/
- https://deploy-preview-30287--kubernetes-io-main-staging.netlify.app/ko/docs/tutorials/stateful-application/basic-stateful-set/#차례대로-파드-생성하기

File: content/ko/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ web-1     1/1       Running   0         18s
 ```
 
 참고로 `web-1` 파드는 `web-0` 파드가 _Running_ ([파드의 단계](/ko/docs/concepts/workloads/pods/pod-lifecycle/#파드의-단계-phase) 참고)
```
Comment (jihoon-seo) [inline content/ko/docs/tutorials/stateful-application/basic-stateful-set.md:124]:
```suggestion
참고로 `web-1` 파드는 `web-0` 파드가 _Running_ ([파드의 단계](/ko/docs/concepts/workloads/pods/pod-lifecycle/#파드의-단계) 참고)
```

File: content/ko/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 2]
```diff
@@ -122,7 +122,7 @@ web-1     1/1       Running   0         18s
...
-및 _Ready_ ([파드의 조건](/ko/docs/concepts/workloads/pods/pod-lifecycle/#파드의-조건-condition)에서 `type` 참고) 상태가 되기 전에 시작하지 않음을 주의하자.
+및 _Ready_ ([파드의 컨디션](/ko/docs/concepts/workloads/pods/pod-lifecycle/#파드의-컨디션-condition)에서 `type` 참고) 상태가 되기 전에 시작하지 않음을 주의하자.
```
Comment (jihoon-seo) [inline content/ko/docs/tutorials/stateful-application/basic-stateful-set.md:125]:
```suggestion
및 _Ready_ ([파드의 컨디션](/ko/docs/concepts/workloads/pods/pod-lifecycle/#파드의-컨디션)에서 `type` 참고) 상태가 되기 전에 시작하지 않음을 주의하자.
```

**Review by jihoon-seo — COMMENTED** [review]
@suhhee1011 기여 감사합니다~~
커멘트를 작성해 보았습니다. 😊
PTAL!

**Comment by seokho-son** [comment]
/assign @jihoon-seo 
신규 컨트리뷰터의 PR, 잘 가이드 해주시길 부탁 드립니다.. :)

**Comment by seokho-son** [comment]
@suhhee1011 
한글화 작업 브랜치가 변경되었습니다.. ^^
dev-1.22-ko.2 -> **dev-1.22-ko.3**

**Comment by ianychoi** [comment]
@suhhee1011 Thank you for your first contribution for Korean localization! Feel free to ask in Korean / English, or you can come to Korean Docs regular meeting if you would like to ask in details. Looking forward to your updates with @jihoon-seo 's guidance thank you both and all

**Comment by jihoon-seo** [comment]
Closing due to [author not responding to comments or feedback in 2 or more weeks](https://kubernetes.io/docs/contribute/participate/pr-wranglers/#when-to-close-pull-requests),
and covered by the PR #31365.

/close

**Pull request closed without merging**